addon/components/course-header.hbs

```diff
@@ -26,16 +26,16 @@
         {{/each}}
       </EditableField>
     {{else}}
-      <h2>
+      <h4>
```

Comment:
I know we don't enforce it everywhere, but the header level here *should* be 2 as it is the critical data on the page.

Reply:
handled.